src/anndata/_io/specs/lazy_methods.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from contextlib import contextmanager
+from pathlib import Path
+
+import h5py
+import numpy as np
+from scipy import sparse
+
+import anndata as ad
+from anndata.compat import H5Array, H5Group, ZarrArray, ZarrGroup
+
+from .registry import _LAZY_REGISTRY, IOSpec
+
+# TODO: settings
+stride = 100
+h5_chunks = 1000
+
+
+def make_dask_array(is_csc, shape, make_dask_chunk, dtype):
```

Comment:
I'm not sure I got a ton from this being its own function as it's not re-used anywhere. Personally wouldn't separate it out.

If it does stick around, I would definitely add type hints.